content/applications/productivity/whatsapp.rst

```diff
@@ -0,0 +1,569 @@
+under :guilabel:`Add an app name`. This name can later be changed in Settings.
+
+.. note::
+   Trademarks and branded elements may not be used in this text section. These include the Meta
+   group of companies. Do not include the word: `WhatsApp` or the system will flag this in error.
+
+Enter the developer email address under the :guilabel:`App contact email` section, about half-way
+down the *details* page.
+
+Under the last section on the *details* page, :guilabel:`Business Account - Optional`, set it to the
+Meta business account using the dropdown option. Click :guilabel:`Create app`. This action will
+create the app and states that the company agrees to the Meta Platform Terms and Developer
+Policies.
+
+Finally enter the Facebook password for security purposes. Click :guilabel:`Submit` to finalize the
+app creation. The browser will redirect to the :guilabel:`Meta for Developers` dashboard.
+
+.. note::
+   If the Meta business account is prohibited from advertizing, claiming an app won't be allowed. To
+   resolve this issue navigate to `<https://business.facebook.com/business>`_ .
+
+   .. seealso::
+      `Meta's documentation on advertizing restrictions
+      <https://www.facebook.com/business/help/975570072950669>`_.
+
+
+Add WhatsApp product to the app
+-------------------------------
+
+Now that the basic structure of the app has been created a product will need to be added to the app.
+To access the Meta App Dashboard go to `<https://developers.facebook.com/apps>`_ and click on the
+app that is being configured.
+
+Since WhatsApp will be used click :guilabel:`Set up` next to the WhatsApp app box towards the bottom
+of the page.
+
+.. seealso::
+   `Meta's WhatsApp developer documentation <https://developers.facebook.com/docs/whatsapp/>`_.
+
+The page will redirect to the configuration page for :guilabel:`WhatsApp Business Platform API`.
+Under :guilabel:`Select a Meta Business Account` use the dropdown to make a selection of the
+preferred business account on this WhatsApp configuration.
+
+Meta will provide a WhatsApp test phone number with 5 test messages. Click :guilabel:`Continue` to
+confirm this Meta Business Account selection. By doing so, the administrator agrees to Meta's terms
+and conditions as linked on the Meta App Dashboard.
+
+Start using the API
+-------------------
+
+Next the administrator will configure the WhatsApp API by adding a phone number and sending a first
+test message. If the browser isn't on the :guilabel:`Quickstart` page for WhatsApp navigate to
+`<https://developers.facebook.com/apps>`_ and click on the app that is being configured. In the left
+menu under :guilabel:`WhatsApp` expand the menu by clicking on the `V` icon.
+
+A small menu will reveal itself with the following options: :guilabel:`Quickstart`, :guilabel:`API
+Setup`, and :guilabel:`Configuration`. Click on :guilabel:`Quickstart` and then click on
```

Comment:
```suggestion
Setup`, and :guilabel:`Configuration`. Click :guilabel:`Quickstart` and then click
```

Reply:
This is where I am stuck by following along. I can't advertise or whatever the error was when I tried to link the business account I made. So from here on out, I can try and follow along but I will mostly have to review for... grammar/clarity/etc because I can't finish setting this up on my own in my test environment.